utils/check_availability.js

```diff
@@ -106,8 +106,9 @@ async function checkRangeAvailability(fromRevision, toRevision) {
  */
 async function checkAndDrawRevisionAvailability(table, name, revision) {
   const promises = [];
+  const DOWNLOAD_HOST = process.env.PUPPETEER_DOWNLOAD_HOST || process.env.npm_config_puppeteer_download_host;
```

Comment:
I wouldn't do anything with this script: it's used solely for the purpose of rolling chromium dependency to the appropriate revision.

Reply:
okay, I thought for the sake of completeness. Will remove it.